lib/assets/stylesheets/utilities/_fonts.scss

```diff
@@ -1,29 +1,47 @@
-/* Warning: this file is automatically generated by `gds/fonts` - do not edit by hand */
```

Comment:
Any idea why this is duplicated from https://github.com/alphagov/tech-docs-gem/blob/master/lib/assets/stylesheets/_fonts.scss?

Reply:
Utilities is the scss file that changes the fonts, the ` /lib/assets/stylesheets/_fonts.scss` looks redundant to me. I will try removing the redundant one and see if anything breaks.

Reply:
Confirmed that the extra font file is redundant. Removed and squashed the commits.